KIPs/kip-82.md

```diff
@@ -0,0 +1,376 @@
+- Ameliorate the risk that block creation depends on larger staker nodes
+- Associate GC reward with the growth of the Klaytn ecosystem
+
+## Specification
+
+Block generation is rewarded for the node operations that are serving as public goods in the Klaytn network. CN and PN are in charge of block creation and propagation while EN is in charge of blockchain data inquiry based on an individual's needs. The infrastructure cost remains similar, enabling some fixed benefits. Therefore, fixed compensation will be equivalent to the sum of the transaction fee and an additional block compensation, which can be both inflationary and deflationary.
+
+The block proposal method will be based on equal selection regardless of the staking amount. The current model provides a block generation reward based on staking amount and the Gini-coefficient. Until now, the Gini coefficient was utilized to alleviate the chance of an entity with more staking amount being selected as a block proposer. As this model provided a fairly equal chance to every participant, each node was promised a certain amount of rewards.
+
+In the new method, the reward for node operation is required since the staking amount is not considered in the block proposer selection. Therefore, a certain percentage of the total reward amount will be placed as a node operation reward.
+
+Node operators will also be compensated with the staking for enhancing network stability from an economic standpoint. The opportunity cost incurred from staking is rewarded based on inflation. In this system, inflation provides a tax-like common effect. Public users can also participate and receive rewards through the GC-run public staking program.
+
+As of September 21st, 2022, of the 300 million KLAY that is minted annually, only 100 million KLAY goes to GC, and 20% and 80% of the distribution is from this 100 million KLAY. The renewal separates GC reward into two groups: basic reward and staked reward. The basic reward is compensating for node operation by providing 20% of GC reward and 100% of gas fee resources from transactions. The staked reward provides only the rest of 80% of the minting resource based on the total stake amount.
+
+The relative ratio of 20% and 80% is a tentative value to be used in the Cypress mainnet. However, this ratio is defined as a governance paramter GC members can change the ratio through the governance process.
+
+![gc reward allocation](../assets/kip-82/gc_reward_allocation.png)
+
+After the Magma hard fork, Klaytn has been burning the first half of the gas fee based on [the KIP-71 proposal](https://kips.klaytn.foundation/KIPs/kip-71) while the second half is distributed. The second half is distributed into GC reward, Klaytn Improvement Reserve (KIR) and Klaytn Growth Fund (KGF).
```

Comment:
Could be good to have a title before this paragraph like:
Change of Gas Fee Distribution?

Reply:
After rephrasing, the paragraphs related to fee burning became too short to make a section.